indico/htdocs/sass/modules/abstracts/_abstract_page.scss

```diff
@@ -206,6 +198,15 @@
 .abstract-review-badges {
     flex-grow: 1;
     color: $gray;
+
+    .js-ratings-toggle {
```

Comment:
No, I'd use the non-js class in JS. The only point of `js-` classes is to make it clear that it's not used for styling - but not the other way round, i.e. a normal class may be used for both.